TrustSDK.xcodeproj/xcshareddata/xcschemes/TrustSDK.xcscheme

```diff
@@ -63,7 +63,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
do we need this change?

Reply:
Needs to set only for one build as well.